stanford_media.post_update.php

```diff
@@ -0,0 +1,355 @@
+<?php
+
+/**
+ * @file
+ * stanford_media.post_update.php
+ */
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+use Drupal\Core\Entity\Entity\EntityViewMode;
+use Drupal\Core\Template\Attribute;
+use Drupal\editor\Entity\Editor;
+use Drupal\embed\Entity\EmbedButton;
+use Drupal\entity_browser\Entity\EntityBrowser;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\filter\Entity\FilterFormat;
+use Drupal\media\Entity\MediaType;
+use Drupal\views\Entity\View;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Convert media browser field widgets to media library widgets.
+ */
+function stanford_media_post_update_8200() {
```

Comment:
Why do you use post `hook_post_update()` instead of `hook_update_N()` From what I know we likely want to have the schema set to 8200. From what I know, this won't do that.

Reply:
it blows up because of the update to the path aliases. If the `hook_update_N` in stanford_media is executed before the update in core, it blows up because of the change of tables

Reply:
we could do one `hook_update_N` to set the schema version. But all the entity updates have to be done in post_update

Reply:
Ah, set up a stub hook_update_8200 should work.